docs/experimental/extendable-param-files.md

```diff
@@ -0,0 +1,63 @@
+# Using the Extendable Bicep Parameters Feature (Experimental!)
+
+## What is it?
+
+Extendable Bicep Parameter Files is a feature that allows you to extend `.bicepparam` files from another `.bicepparam` file in order to reuse parameters across multiple deployments.
+
+When using extendable bicep parameter files, you will have a `base.bicepparam` file that can be used by multiple `.bicep` and `.bicepparam` files.
+
+The `base.bicepparam` file will link to a `shared.bicepparam` file, which will refer to a `main.bicep` file and your `base.bicepparam file`. In this case, you will be able to reuse and refer to this `base.bicepparam` file multiple times.
+
+## Using
+
+`main.bicep` This is your main bicep file, which will define your parameters for deployment.
+
+```bicep
+param foo string
+param foo string
+```
+
+`base.bicepparam` This is your base bicepparam file, which can be reused by multiple shared.bicepparam files and in multiple deployments.
+
+```bicep
+using none
+
+param foo = 'foo'
+param bar = 'bar'
+```
+
+`shared.bicepparam` This is your shared bicepparam file, which will refer to one main.bicep file and one base.bicepparam file. Any parameter definition in this file will override all previous definitions.
```

Comment:
"Any parameter definition" - are these parameter _definitions_ or a parameter value? I think of the parameter definition as what is declared in the `.bicep` file.

Reply:
I guess I wasn't sure about usage here. I personally assumed that defining a parameter value is when you assign a new value, but I can change it to just parameter values if that's clearer!